src/ui/features/referral-program/SuccessDialog/SuccessDialog.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import { UIText } from 'src/ui/ui-kit/UIText';
+import { DialogTitle } from 'src/ui/ui-kit/ModalDialogs/DialogTitle';
+import { VStack } from 'src/ui/ui-kit/VStack';
+import { Button } from 'src/ui/ui-kit/Button';
+import RocketSrc from 'src/ui/assets/rocket.png';
+import Rocket2xSrc from 'src/ui/assets/rocket@2x.png';
+import SuperheroSrc from 'src/ui/assets/superhero.png';
+import Superhero2xSrc from 'src/ui/assets/superhero@2x.png';
+import PremiumIcon from 'jsx:src/ui/assets/premium.svg';
+import ChartIcon from 'jsx:src/ui/assets/chart.svg';
+import StarIcon from 'jsx:src/ui/assets/star.svg';
+import CsvIcon from 'jsx:src/ui/assets/csv.svg';
+import { focusNode } from 'src/ui/shared/focusNode';
+import type { ReferrerData } from 'src/modules/zerion-api/requests/check-referral';
+import { ReferrerLink } from '../shared/ReferrerLink';
+import { PremiumTrialBanner } from '../shared/PremiumTrialBanner';
+import { FeatureCard } from '../shared/FeatureCard';
+import * as styles from './styles.module.css';
+
+export function SuccessDialog({
+  referrer,
+  onDismiss,
+}: {
+  referrer: Pick<ReferrerData, 'handle' | 'address'>;
+  onDismiss: () => void;
+}) {
+  return (
+    <VStack gap={24} className={styles.successDialog}>
```

Comment:
This [comment](https://github.com/zeriontech/zerion-wallet-extension/pull/629/files#r1810570851) is probably relates to this component

Reply:
✅